sam-app/lambda_functions/template.yaml

```diff
@@ -125,39 +125,39 @@ Parameters:
       - ERROR
       - CRITICAL
   PostcallRecordingImportEnabled:
-    Default: true
+    Default: 'true'
     Description: Set to false if importing call recordings into Salesforce should not be enabled on the package level.
     Type: String
     AllowedPattern: ^(true|false)$
     AllowedValues:
     - true
     - false
   PostcallTranscribeEnabled:
-    Default: true
-    Description: Set to false if post-call transcription should not be enabled on the package level. 
+    Default: 'true'
+    Description: Set to false if post-call transcription should not be enabled on the package level [true].
```

Comment:
I'm curious, why do put this `[true]` in the description?

Reply:
you are correct. that's not necessary. Let me remove.

Reply:
> you are correct. that's not necessary. Let me remove.

and updated.